code/game/objects/items/storage/boxes.dm

```diff
@@ -381,6 +381,12 @@
 	spawn_number = 15
 	spawn_type = /obj/item/explosive/grenade/phosphorus
 
+/obj/item/storage/box/nade_box/tanglefoot
+	name = "\improper M40-T gas grenade box"
+	desc = "A secure box holding 25 M40-T gas grenades. 100% safe to use around masked marines."
+	icon_state = "marinebox"
+	spawn_type = /obj/item/explosive/grenade/drainbomb
+
```

Comment:
make it hold less please

Reply:
Already discussed this in discord. Marines get 25 HE nades and incin nades/box, and this is, if anything, less effective than those as it's only gas that drains plasma.